force-app/main/default/classes/SOQL.cls

```diff
@@ -344,18 +469,30 @@ public with sharing class SOQL {
      * Example: setPrimaryContact() should set the 'Primary_Contact__c' field for every Opportunity in the org
      */
     public static void setPrimaryContact() {
-        //Use the getOpportunityDetailsForClosedWon() method to get the Opportunities
+        List<Opportunity> listOppsForPrimC = SOQL.getOpportunityDetailsForClosedWon();//Use the getOpportunityDetailsForClosedWon() method to get the Opportunities
```

Comment:
No need to call the class here. since you're already operating in the SOQL class